src/SharePrefix.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Storj\Uplink;
+
+use FFI;
+use Storj\Uplink\Internal\Scope;
+use Storj\Uplink\Internal\Util;
+
+/**
+ * Parameters to create an Access grant
+ */
+class SharePrefix
+{
+    private string $bucket;
+
+    /**
+     * Prefix of the accessible object keys.
+     *
+     * Within a bucket, the hierarchical key derivation scheme is
+     * delineated by forward slashes (/), so encryption information will be
+     * included in the resulting access grant to decrypt any key that shares
+     * the same prefix up until the last slash.
+     */
+    private string $prefix;
+
+    public function __construct(string $bucket, string $prefix)
```

Comment:
Can we make the `$prefix` param optional?

Reply:
Done